crates/flox/src/commands/environment.rs

```diff
@@ -452,14 +452,29 @@ impl Uninstall {
         let mut environment =
             resolve_environment(&flox, self.environment.as_deref(), "uninstall").await?;
 
-        environment
+        let packages_str = packages_to_string(&packages);
+        let plural = packages.len() > 1;
+
+        if environment
             .uninstall(
                 packages.drain(..),
                 &flox.nix(Default::default()),
                 &flox.system,
             )
             .await
-            .context("could not uninstall packages")?;
+            .context("could not uninstall packages")?
+        {
+            println!(
+                "🗑️ Uninstalled {packages_str} from '{}' environment.",
+                environment.environment_ref()
+            );
+        } else {
+            let verb = if plural { "are" } else { "is" };
+            println!(
```

Comment:
```suggestion
            info!(
```

Reply:
Should we be using `info!` a bit more consistently? E.g. in init?

Reply:
yes, we should I think that's the right verbosity level.
we should use println for the cases that we expect are read from stdout, e.g. json outputs maybe list(?).